install/configure_administrative_access.md

```diff
@@ -83,11 +83,12 @@ BMC/controller passwords.
 
       The Olympus hardware (NodeBMCs, ChassisBMCs, RouterBMCs) needs to have recovery firmware loaded to the cray-tftp server in case the BMC loses its firmware. The BMCs are configured to load a recovery firmware from a TFTP server. This procedure does not modify any BMC firmware, but only stages the firmware on the TFTP server for download in the event it is needed.
 
+      This step requires the CMS software, Cray CLI, and HPC Firmware Pack (HFP) to be installed.  If these are not currently installed, this step will need to be skipped and run later in the install process.
```

Comment:
```suggestion
      This step requires the CSM software, Cray CLI, and HPC Firmware Pack (HFP) to be installed. If these are not currently installed, then this step will need to be skipped and run later in the install process.
```

Reply:
Thanks - updated

Reply:
1.2 and master PRs will be coming after finalizing these updates and PR 1459 and 1460 have been merged.